src/dashboard/src/main/management/commands/rebuild_aip_index_from_storage_service.py

```diff
@@ -131,32 +131,36 @@ def handle(self, *args, **options):
         if options["uuid"]:
             package_uuid = options["uuid"]
             self.info("Reindexing package {}".format(package_uuid))
-            package_info = storageService.get_file_info(uuid=package_uuid)
+            try:
+                package_info = storageService.get_file_info(uuid=package_uuid)[0]
+            except IndexError:
+                self.error(
+                    "Error indexing package {0}. "
+                    "Package with supplied UUID not found in Storage Service".format(
+                        package_uuid
+                    )
+                )
+                return
```

Comment:
As the return from this function will be `None`. We should consider removing the explicit return here.

Reply:
The return will be `None`, but we need this return so that the task only indexes the one package we requested with the `--uuid` tag. Otherwise, the task will index this one package, and then move on to indexing all AIPs.